examples/web-chat/.cspell.json

```diff
@@ -3,6 +3,7 @@
   "$schema": "https://raw.githubusercontent.com/streetsidesoftware/cspell/master/cspell.schema.json",
   "language": "en",
   "words": [
+    "AOGECG2SPND25EEFMAJ5WF3KSGJNSGV356DSTL2YVLLZWIV6SAYBM",
```

Comment:
just add `enrtree://` in the patterns to ignore.

Reply:
https://github.com/waku-org/js-waku-examples/pull/278